ios/Surveys/Resources/Assets/Assets.xcassets/splashBackground.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "Background.pdf",
+      "filename" : "Image (1).pdf",
```

Comment:
Nit: the name of the image file is a little bit weird, I think we can choose a legal one.